cmd/broker/ingress/main.go

```diff
@@ -17,12 +17,18 @@ limitations under the License.
 package main
 
 import (
-	"github.com/google/knative-gcp/pkg/broker/ingress"
 	"go.uber.org/zap"
+
+	"github.com/google/knative-gcp/pkg/broker/ingress"
 	"knative.dev/pkg/logging"
 	"knative.dev/pkg/signals"
 )
 
+// main creates and starts an ingress handler using default options.
+// 1. It listens on port 8080
```

Comment:
It should listen on `PORT` if that env var is present, since that's the env var serving uses.